balancer/frontend-v2
# Add original error info for Sentry

# Description

In some Sentry error logs (e.g. https://balancer-labs.sentry.io/issues/4173315674/?project=5725878) there is a `data.originalError` value (which I believe is the EVM error) that is not being parse correctly because it's too deep. We should surface this in the additional data. 

I also found https://github.com/enzoferey/ethers-error-parser which could be useful but I worry what happens if we attempt to parse a non ethers error with it. 

## Type of change


## How should this be tested?

- Create a transaction that fails (can do this via disconnecting your hardware device) and debug the code to ensure originalError is received and passed. 
- Ensure this does not interfere with other error types that don't have `data.originalError` (not sure how)

## Visual context

Please provide any relevant visual context for UI changes or additions. This could be static screenshots or a loom screencast.

## Checklist:
